### DISCUSSION TITLE
How to select methods for a custom operator

### DISCUSSION THREAD

**Discussion by vijaya-lakshmi-venkatraman** [apache/airflow#19887] [Q&A]
Hello,
I am a newbie to Airflow trying to create a new custom operator for a document database.  
I would like to know how to select the methods that should be included in the hook.
Should it be all CRUD operations?

Please can someone help point to a documentation if there is one already available?
If not, please can you provide some pointers on how this should be done?

**Comment by potiuk** [comment]
It's up to you, what subset you choose. There are no guidelines. You should think as a user who wants to write some workflows with. Come up with several "example DAGs" you will want to support and the list of methods should come from it.

Start small. You can always extend it later. It's never "done"
